src/AnalyzeCli.cpp

```diff
@@ -54,6 +56,15 @@ static void report(ReportFormat format, const char* name, const Luau::Location&
         // Note: GNU end column is inclusive but our end column is exclusive
         fprintf(stderr, "%s:%d.%d-%d.%d: %s: %s\n", name, loc.begin.line + 1, loc.begin.column + 1, loc.end.line + 1, loc.end.column, type, message);
         break;
+
+    case ReportFormat::Prettier:
+        // Use Reporter.hpp to create new reporter error
+        auto file = new reporter::SimpleFile(name);
+        auto err = reporter::Error(
+                type, message,
```

Comment:
I wonder if the top line should include the error message, rather than just `TypeError` - i think that should go as part of the code.